<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpUdpComTool</class>
 <widget class="QWidget" name="TcpUdpComTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pushButtonStart">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>50</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEditRec">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>481</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEditSend">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>481</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxStyle">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="frameStatus">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>458</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="labelStatusTag">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>当前状态:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelStatus">
    <property name="geometry">
     <rect>
      <x>66</x>
      <y>0</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>未连接</string>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="comboBoxClientList">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>90</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>180</y>
     <width>141</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="radioButton16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>4</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>16进制</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonString">
    <property name="geometry">
     <rect>
      <x>73</x>
      <y>4</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>字符串</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSend">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>ctrl+Enter</string>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="checkBoxHexRec">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>460</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Hex接收显示</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxLocal">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>300</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>本地</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLineEdit" name="lineEditPortLocal">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>44</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxLocal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>16</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxAim">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>380</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>目的</string>
   </property>
   <widget class="QLineEdit" name="lineEditPortAim">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>45</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditIpAim">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QToolButton" name="toolButtonShortCut">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>453</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>快捷键</string>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClearRec">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>270</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>清空接收框</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboComNum">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>122</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxBaud">
   <property name="geometry">
    <rect>
     <x>572</x>
     <y>123</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
